### PR TITLE
Move ohnoes images to creatures folder

### PR DESCRIPTION
**Pull request by MattMS**
This is a start at fixing #609.

I cannot see the images in the image picker on my local demo, so I am guessing there are references that I have missed updating.

### CODE REVIEW

**Comment by Gigabyte-Giant** [comment]
Unless I'm terribly mistaken, I do believe that in order for images to show up in the image picker, they need to be added to [`js/shared/images.js`](https://github.com/Khan/live-editor/blob/master/js/shared/images.js#L9-L30) as well.

**Comment by MattMS** [comment]
Added and ran `node build-all-images.js` after but they are still not showing up.

**Comment by MattMS** [comment]
Actually, mostly seems good after running `gulp scripts`.
I can see and select both of those images in the preview area.

*But* it's now not showing the images under the preview.
So I need to find where that is being specified and update the code.

**Comment by MattMS** [comment]
Latest commit manually fixes the path and now the images appear to be working.

File: build/tmpl/live-editor.js [hunk 1]
```diff
@@ -115,12 +115,12 @@ function program21(depth0,data) {
...
   if(typeof stack1 === functionType) { stack1 = stack1.call(depth0, { hash: {} }); }
   else if(stack1=== undef) { stack1 = helperMissing.call(depth0, "imagesDir", { hash: {} }); }
-  buffer += escapeExpression(stack1) + "/ohnoes-happy.png\"/>\n                </div>\n                <a class=\"error-buddy-thinking\" style=\"display:none;\" href=\"javascript:void()\">\n                    <img src=\"";
+  buffer += escapeExpression(stack1) + "/creatures/ohnoes-happy.png\"/>\n                </div>\n                <a class=\"error-buddy-thinking\" style=\"display:none;\" href=\"javascript:void()\">\n                    <img src=\"";
   foundHelper = helpers.imagesDir;
   stack1 = foundHelper || depth0.imagesDir;
   if(typeof stack1 === functionType) { stack1 = stack1.call(depth0, { hash: {} }); }
   else if(stack1=== undef) { stack1 = helperMissing.call(depth0, "imagesDir", { hash: {} }); }
-  buffer += escapeExpression(stack1) + "/ohnoes-hmm.png\"/>\n                    ";
+  buffer += escapeExpression(stack1) + "/creatures/ohnoes-hmm.png\"/>\n                    ";
```
Comment (kevinbarabash) [inline build/tmpl/live-editor.js]: Why is `ohnoes-hmm.png` lowercase here but uppercase in `live-editor.handlebars`?

Comment (MattMS) [inline build/tmpl/live-editor.js]: Nice catch! I'll fix that up now.

**Comment by kevinbarabash** [comment]
@MattMS are you editing `build/tmpl/live-editor.js` manually?  Running `gulp build` should update it automatically.  Is that not the case?

**Comment by MattMS** [comment]
I built the templates with `gulp templates` and the scripts with `gulp scripts` and then only committed the significant changes.

Running `gulp build` gives me a bunch of extra rubbish that I think is due to my Git setup and also some Windows-specific issues (line endings) in templates.

That said, running `gulp build` again showed me that I had missed a change that I'm syncing to this PR now.

**Comment by MattMS** [comment]
The build seems to be failing (in Travis CI) but from a quick glance it seems to be because of phantomjs not installing.

**Comment by MattMS** [comment]
Rebased from master and all the tests passed, so it must have been Travis messing up last time.

Is there anything more this PR requires?

**Comment by kevinbarabash** [comment]
@MattMS I'm happy to see the tests are passing.  Thanks for rebasing.

**Pull request merged**